linters/eslintrc.json

```diff
@@ -1706,5 +1706,15 @@
         ]
       }
     ]
+  },
+  "parser": "babel-eslint",
+  "overrides": {
+    "files": [
+      "*.test.js",
+      "**/setupTets.js"
```

Comment:
same here unless this file name has this typo. in that case, that should probably also be fixed